test/parallel/test-repl-definecommand.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+require('../common');
+
+var stream = require('stream'),
+    assert = require('assert'),
+    repl = require('repl');
+
```

Comment:
at least the requires for sure. make sure it passes lint. the rest I'd consider optional